modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/RunInterruptMonitor.java

```diff
@@ -69,6 +78,28 @@ public void run() {
         }
     }
 
+    private List<RunInterruptEvent> getTimedOutInterruptEvents() throws FrameworkException {
```

Comment:
Comment on what this does might be good ? Not obvious what an interruptEvent is.
Or a different name ?

getInterruptedTestsWhichShouldHaveCleanedUpByNow or something. OK that's rediculous, but might be worth thinking of a better name ?

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/376/commits/18fe87da32d39dda21609f1eb11d8343a304fbfa